capi/geos_ts_c.cpp

```diff
@@ -1064,6 +1065,32 @@ extern "C" {
         });
     }
 
+    Geometry*
+    GEOSExtent_r(GEOSContextHandle_t extHandle, Geometry** geoms, unsigned int ngeoms)
+    {
+        return execute(extHandle, [&]() {
+            Envelope extent;
+            const Geometry* geom;
+            for (std::size_t i = 0; i < ngeoms; i++) {
+                geom = geoms[i];
```

Comment:
This probably needs 
```
if (geom == NULL) {
    continue
}
```
(otherwise `geom->` calls below would break)

May also want to combine that with a check for empty too in order to skip over empty geometries.